embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/anr/AnrOtelMapper.kt

```diff
@@ -67,12 +69,12 @@ internal class AnrOtelMapper(
             attrs.add(Attribute("sample_code", it.toString()))
         }
         sample.threads?.singleOrNull()?.let { thread ->
-            attrs.add(Attribute("thread_state", thread.state.toString()))
+            attrs.add(Attribute(JvmAttributes.JVM_THREAD_STATE.key, thread.state.toString()))
```

Comment:
Does the value here match the definition in the semantic conventions? I want to make sure what we are writing is what's expected - otherwise, we should keep using our custom one.

Reply:
yeah, I think so. Sample values include `runnable` and `blocked`, and those are part of the states in `Thread.java` (we get the values from there)